src/main/java/com/exactpro/epfast/template/ByteVectorField.java

```diff
@@ -0,0 +1,13 @@
+package com.exactpro.epfast.template;
+
+public interface ByteVectorField extends Instruction {
+
+    IdentityRef getFieldId();
```

Comment:
the return type should be `Identity`. It's true for all fields. Probably it makes sense to create `FieldInstruction` or `Field` interface with this property